estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,34 @@
+from odoo import api, fields, models
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Property Offer"
+
+    price = fields.Float()
+    status = fields.Selection(
+        string="Type",
+        selection=[("accepted", "Accepted"), ("refused", "Refused")],
+        copy=False,
+    )
+    partner_id = fields.Many2one("res.partner", required=True)
+    property_id = fields.Many2one("estate.property", required=True)
+    validity = fields.Integer(default=7)
+    date_deadline = fields.Date(
+        compute="_compute_date_deadline", inverse="_inverse_date_deadline"
+    )
+
+    @api.depends("validity")
```

Comment:
All properties referenced in the computation logic should be part of the dependencies
```suggestion
    @api.depends("validity", "create_date")
```